frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/storage-tab-initial-state.ts

```diff
@@ -186,11 +190,8 @@ export const getNewProvisionSourceStorage = (state: any, id: string): VMWizardSt
     const iCommonTemplates = iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates);
     const iTemplate =
       iCommonTemplates && iGetRelevantTemplate(null, iCommonTemplates, relevantOptions);
-    const pvcName = iGetAnnotation(
-      iTemplate,
-      `${TEMPLATE_DATAVOLUME_ANNOTATION}/${relevantOptions.os}`,
-    );
-    const pvcNamespace = iGetAnnotation(iTemplate, `${TEMPLATE_DATAVOLUME_ANNOTATION}/namespace`);
+    const pvcName = iGetPrameter(iTemplate, TEMPLATE_DATAVOLUME_NAME_PARAMETER).value;
```

Comment:
is the parameter returned really a plain JS object?

Reply:
:-) no, I'm fixing that :+1: